walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/handlers/OpenId4VCIRequestHandler.kt

```diff
@@ -260,4 +261,12 @@ internal class OpenId4VCIRequestHandler(
             return credentialIssuer + suffix
         return credentialIssuer
     }
+
+    private suspend fun preValidatePinRequirement(openId4VCIPinRequirement: OpenId4VCIPinRequirement) {
+        OpenID4VCIPreAuthAccessTokenResolver(libraryConfiguration).resolve(
+            openId4VCIPinRequirement.preAuthorizedCode,
+            openId4VCIPinRequirement,
+            openId4VCIPinRequirement.accessTokenEndpoint!!
```

Comment:
At this point, the requirement should have an accessTokenEndpoint. But if it is missing, we need to fail. Currently, it will always be there, but it is missed part of any future work like refactoring, it is a good idea to fail the flow since it is a required field.